src/tests/component/Auth.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Auth from '../../components/Auth/Auth';
+
+describe('<Auth />', () => {
+  test('should render auth', () => {
+    const wrapper = shallow(<Auth />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  test('should render auth', () => {
+    const wrapper = shallow(<Auth />);
+    const auth = 'signup';
+    const { switchAuth } = wrapper.instance();
+    switchAuth(auth);
+    expect(wrapper.state().auth).toEqual(auth);
+  });
+
+  test('should render auth', () => {
+    const wrapper = shallow(<Auth />);
+    const spy = jest.spyOn(wrapper.instance(), 'switchAuth');
+    const signInBtn = wrapper.find('[data-test="signin-btn"]');
+    signInBtn.simulate('click');
+    expect(spy).toHaveBeenCalled();
+  });
+
+  test('should render auth', () => {
+    const wrapper = shallow(<Auth />);
+    const spy = jest.spyOn(wrapper.instance(), 'switchAuth');
+    const signUpBtn = wrapper.find('[data-test="signup-btn"]');
+    signUpBtn.simulate('click');
+    expect(spy).toHaveBeenCalled();
+  });
+});
```

Comment:
All the test suits in this file have the same description `should render auth` please change it.

Reply:
yeah, sorry had been copying and pasting it down then changing the test case, forgot to change that.